packages/web/src/services/audius-backend/Jupiter.ts

```diff
@@ -146,7 +147,12 @@ const getSwapInstructions = async ({
       } as TransactionInstruction
     })
   return {
-    instructions: instructionsFlattened,
+    instructions: [
+      ...instructionsFlattened,
+      ComputeBudgetProgram.setComputeUnitPrice({
```

Comment:
Doesn't there already exist a compute budget instr?

Reply:
I removed it above and added here because the filter/map above doesn't work with it

Reply:
Oh wait I see

Reply:
It already exists though. Jupiter returns one. See line 126

Reply:
sorry i missed this in the earlier review!

Reply:
Are we sure that includes a priority fee though?

Reply:
Looks like we would need to call jup with a microLamports specified here https://station.jup.ag/api-v6/post-swap-instructions

Reply:
yeah it doesn't unless you do that - but if you do both it'll fail:

> Caution: Transactions can only contain one of each type of compute budget instruction. Duplicate types will result in an [TransactionError::DuplicateInstruction](https://github.com/solana-labs/solana/blob/master/sdk/src/transaction/error.rs#L144-145) error, and ultimately transaction failure.

https://solana.com/docs/intro/transaction_fees

So either replace theirs with our own or put that in the request I think

Reply:
It works though 🤔  anyway, here's a better change: https://github.com/AudiusProject/audius-protocol/pull/7851